Car/TeamNull SmartCar/src/util/HardwareUtil.cpp

```diff
@@ -4,17 +4,63 @@
 unsigned long BLINK_INTERVAL = 200;
 
 VL53L0X frontSensor;
-// VL53L0X rearSensor;
+VL53L0X rearSensor;
 
 const int frontSensorPin = 33;
 const int rearSensorPin = 32;
 
+const int FR_TRIGGER_PIN = 19;
+const int FR_ECHO_PIN = 23;
+const int FL_TRIGGER_PIN = 5;
+const int FL_ECHO_PIN = 18;
+const unsigned int MAX_DISTANCE = 300;
+
 BrushedMotor leftMotor(smartcarlib::pins::v2::leftMotorPins);
 BrushedMotor rightMotor(smartcarlib::pins::v2::rightMotorPins);
 DifferentialControl control(leftMotor, rightMotor);
 
+SR04 frontRight(FR_TRIGGER_PIN, FR_ECHO_PIN, MAX_DISTANCE);
+SR04 frontLeft(FL_TRIGGER_PIN, FL_ECHO_PIN, MAX_DISTANCE);
+
 SimpleCar car(control);
 
+// //To find pulses per meter...
+// #include <Smartcar.h>
+
+// const unsigned long LEFT_PULSES_PER_METER  = 100;
+// const unsigned long RIGHT_PULSES_PER_METER = 100;
+// DirectionlessOdometer leftOdometer(
+//     smartcarlib::pins::v2::leftOdometerPin,
+//     []() { leftOdometer.update(); },
+//     LEFT_PULSES_PER_METER);
+// DirectionlessOdometer rightOdometer(
+//     smartcarlib::pins::v2::rightOdometerPin,
+//     []() { rightOdometer.update(); },
+//     RIGHT_PULSES_PER_METER);
+// void setup() {
+//     Serial.begin(9600);
+// }
+// void loop() {
+//     // Manually roll the vehicle for one meter
+//     // and note down the printed out value. That is the
+//     // correct pulses:centimeters ratio for your odometer
+//     // that you should use when initializing the odometer
+//     // in its constructor
+//     Serial.print("Left: ");
+//     Serial.println(leftOdometer.getDistance());
+//     Serial.print("Right: ");
+//     Serial.println(rightOdometer.getDistance());
+
+//     delay(100);
+// }
```

Comment:
If you aren't using some code just remove it

Reply:
Removed (in my branch)